packages/react/src/components/Tabs/Tabs.js

```diff
@@ -35,6 +35,11 @@ export default class Tabs extends React.Component {
      */
     hidden: PropTypes.bool,
 
+    /**
+     * Provide the props that describe the left overflow button
+     */
+    leftOverflowButtonProps: PropTypes.object,
```

Comment:
Is there a way we could allow folks to configure the labels without giving them a way to pass in any prop to the buttons? Maybe something like `overflowStartLabel`, `overflowEndLabel`? or `scrollPrevLabel`, `scrollMoreLabel` (just throwing out some ideas)

Exposing that this is a button and allowing folks to configure it might expose some internal implementation details, or prevent us from refactoring like if we wanted the label to be inside of the button instead of `aria-label`, or would want to connect it using `aria-labelledby` or something 🤔

Reply:
yeah I was looking ahead and figuring that people would want to extend the button so I didn't want to make the prop too restrictive by making it label-only

the use of `aria-label` was suggested by @carmacleod since there is no visible label for these overflow buttons

Reply:
If we're anticipating folks to configure the buttons directly it might be helpful to extract them as components and allow folks to pass props directly to them (and support providing these buttons as children)

Definitely think `aria-label` is the right call, just was saying that this would restrict changes we make if we ever want to change how these buttons are implemented (e.g. if we wanted to use `aria-labelledby` for some reason). 

An example of this could be the components that forward props to downshift, for example. It makes updating that package difficult as we have to manage the breaking changes in props (like name changes) if we want to update it in a minor release.

Reply:
I just don't know that we need to provide that much configurability for these buttons when the logic that displays them is all internal. is this a future change that you are predicting for this component? it seems like we're solving an issue that isn't occurring given the component spec and a11y requirements